### PR TITLE
Fix typos on word development and on some Brazilian Portuguese translations

### PR DESCRIPTION
**Pull request by jaimeadf**

### CODE REVIEW

**Comment by ImagicTheCat** [comment]
- A translation fix doesn't respect the capitalization of words.
- The `docs/` files are generated and shouldn't be modified manually, but it should work in this case.

**Comment by jaimeadf** [comment]
Oh sorry, I didn’t about that. If you want I can force push commits without the doc changes or close this PR and create another one.

**Comment by ImagicTheCat** [comment]
You can rebase the PR as you wish. Except for the uppercase issue, it should be fine because I think you modified the html as the generation would, it's just not a good idea to do that for more complex edits.

File: vrp/cfg/lang/pt-br.lua [hunk 1]
```diff
@@ -254,9 +254,9 @@ local lang = {
     missing = "~r~Faltando {2} {1}.",
     full = "~r~Inventário cheio.",
     chest = {
-      title = "Báu",
-      already_opened = "~r~Este báu já foi aberto por alguém.",
-      full = "~r~Báu cheio.",
+      title = "baú",
```
Comment (ImagicTheCat) [inline vrp/cfg/lang/pt-br.lua:257]: Here, it's lowercase instead of uppercase.

**Comment by jaimeadf** [comment]
Thanks, I’ll make all those changes in about an hour or two.

File: vrp/cfg/lang/pt-br.lua [hunk 2]
```diff
@@ -254,9 +254,9 @@ local lang = {
...
+      already_opened = "~r~Este baú já foi aberto por alguém.",
+      full = "~r~baú cheio.",
```
Comment (jaimeadf) [inline vrp/cfg/lang/pt-br.lua:259]: I also need to capitalize here.

**Comment by ImagicTheCat** [comment]
Thanks, it would be nice to rebase all the commits to one or two meaningful ones (for clarity).

**Comment by jaimeadf** [comment]
Wouldn't be better for you to use the squash option to merge them?
The squash merges all commits into one and lets you change its message as your liking.

image

**Comment by jaimeadf** [comment]
I squashed them for you, sorry for the inconvenience.

**Comment by ImagicTheCat** [comment]
I prefer to let people do it first because that's what I do myself when I'm done, I cleanup/clarify the commits. Thanks for the fixes.

**Pull request merged**